app/translations/si.txt

```diff
@@ -0,0 +1,334 @@
+
+WINDOW_TITLE=එක්ස්ඩීඑම් 2018
+ALL_DOWNLOADS=සියල්ල
+ALL_UNFINISHED=අසම්පූර්ණ
+ALL_FINISHED=සම්පූර්ණ
+CAT_ALL=සියලුම ප්‍රවර්ග
+CAT_DOCUMENTS=ලේඛන
+CAT_COMPRESSED=Compressed
+CAT_MUSIC=සංගීතය
+CAT_VIDEOS=දෘශ්‍යක
+CAT_PROGRAMS=ක්‍රමලේඛන
+BROWSER_MONITORING=අතිරික්සුව නිරීක්ෂකය
+MENU_FILE=ගොනුව
+MENU_DOWNLOAD=බාගැනීම්
+MENU_TOOLS=මෙවලම්
+MENU_HELP=උපකාර
+MENU_ADD_URL=ඒ.ස.නි.(URL) එකතු කරන්න
+MENU_BATCH_DOWNLOAD=සමුදායක් බාගැනීම
+MENU_CLIP_ADD=පසුරුපුවරුව නිරීක්‍ෂණය
+MENU_VIDEO_DWN=දෘශ්‍යකය බාගත කරන්න
+MENU_DELETE_DWN=Delete downloads
+MENU_DELETE_COMPLETED=අවසාන වූ දෑ හිස්කරන්න
+MENU_IMPORT=ආයාත කරන්න
+MENU_EXPORT=නිර්යාත කරන්න
+MENU_EXIT=පිටවන්න
+MENU_PAUSE=Pause
+MENU_RESUME=යළි පටන්ගන්න
+MENU_RESTART=යළි අරඹන්න
+MENU_START_Q=පෝලිම අරඹන්න
+MENU_STOP_Q=පෝලිම_නවත්වන්න
+MENU_OPTIONS=විකල්ප...
+MENU_REFRESH_LINK=නැවුම් කිරීමේ සබැඳිය
+MENU_PROPERTIES=ගුණාංග
+MENU_FORCE_ASSEMBLE=එකලාසයට බලකරන්න
+MENU_SPEED_LIMITER=වේග සීමාකාරකය
+MENU_MEDIA_CONVERTER=Media converter
+MENU_BROWSER_INT=අතිරික්සු නිරීක්ෂකය
+MENU_CONTENTS=අන්තර්ගතයන්...
+MENU_HOME_PAGE=මුල් පිටුව
+MENU_UPDATE=යාවත්කාල සඳහා පිරික්සන්න
+MENU_ABOUT=එක්ස්ඩීඑම් ගැන...
+SORT_DATE=දිනය
+SORT_SIZE=ප්‍රමාණය
+SORT_NAME=නම
+SORT_TYPE=වර්ගය
+SORT_ASC=ආරෝහණ
+SORT_DESC=අවරෝහණ
+SORT_DATE_DESC=නවතම ඉහළින්
+SORT_DATE_ASC=පරණම ඉහළින්
+SORT_SIZE_DESC=විශාලම ඉහළින්
+SORT_SIZE_ASC=කුඩාම ඉහළින්
+SORT_NAME_DESC=නම [ෆ-අ]
+SORT_NAME_ASC=නම [අ-ෆ]
+SORT_TYPE_DESC=වර්ගය [ෆ-අ]
+SORT_TYPE_ASC=වර්ගය [අ-ෆ]
+STAT_PAUSED=Paused
+STAT_FINISHED=අවසන් වී ඇත
+STAT_DOWNLOADING=බාගත වෙමින්
+STAT_ASSEMBLING=එකලස් වෙමින්
+ND_TITLE=නව බාගැනීම
+ND_ADDRESS=ලිපිනය
+ND_FILE=ගොනුව
+ND_MORE=තව...
+ND_CANCEL=අවලංගු කරන්න
+ND_DOWNLOAD_NOW=දැන් බාගන්න
+ND_DOWNLOAD_LATER=පසුව බාගත කරන්න
+ND_IGNORE_URL=Do not capture download from this address
```

Comment:
මෙම ලිපිනයෙන් බාගත කිරීම අල්ලා නොගන්න =Do not capture download from this address